src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -692,4 +718,56 @@ class SVTResourceResolver: AzSKRoot {
+
+    # getting resources count and sending them to telemetry as well
+    [void] GetResourceCount($projectName, $organizationId, $projectId, $projectData) {
+        try{
+            # fetching the repository count of a project
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/git/repositories?api-version=6.0"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            $projectData['repositories'] = ($responseList | Measure-Object).Count
+
+            # fetching the testPlan count of a project
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/testplan/plans?api-version=6.0-preview.1"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            $projectData['testPlan'] = ($responseList | Measure-Object).Count
+
+            # fetching the taskGroups count of a project
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/distributedtask/taskgroups?api-version=6.0-preview.1"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            $projectData['taskGroups'] = ($responseList | Measure-Object).Count
+
+            # fetch the builds count
+            $resourceURL = ("https://dev.azure.com/{0}/{1}/_apis/build/definitions?api-version=4.1&queryOrder=lastModifiedDescending&`$top=10000") -f $($this.SubscriptionContext.SubscriptionName), $projectName;
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL);
+            $projectData['build'] = ($responseList | Measure-Object).Count
+
+            # fetch the release count
+            $resourceURL = ("https://vsrm.dev.azure.com/{0}/{1}/_apis/release/definitions?api-version=4.1-preview.3&`$top=10000") -f $($this.SubscriptionContext.SubscriptionName), $projectName;
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL);
+            $projectData['release'] = ($responseList | Measure-Object).Count;
+
+            # fetch the agent pools count
+            if($projectData["agentPools"] -eq -1) {
+                $agentPoolsDefnURL = ("https://dev.azure.com/{0}/{1}/_settings/agentqueues?__rt=fps&__ver=2") -f $($this.SubscriptionContext.SubscriptionName), $projectName;
+                $agentPoolsDefnsObj = [WebRequestHelper]::InvokeGetWebRequest($agentPoolsDefnURL);
+                if (([Helpers]::CheckMember($agentPoolsDefnsObj, "fps.dataProviders.data") ) -and (($agentPoolsDefnsObj.fps.dataProviders.data."ms.vss-build-web.agent-queues-data-provider") -and $agentPoolsDefnsObj.fps.dataProviders.data."ms.vss-build-web.agent-queues-data-provider".taskAgentQueues)) {
+                    $taskAgentQueues = $agentPoolsDefnsObj.fps.dataProviders.data."ms.vss-build-web.agent-queues-data-provider".taskAgentQueues;
+                    $projectData["agentPools"] = ($taskAgentQueues | Measure-Object).Count
```

Comment:
Service connection is not added? like other resources count added here

Reply:
It is being evaluated at line 430, so no need to add here